Web/starter_project_iconics/styles/singleBlog.css

```diff
@@ -0,0 +1,50 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=IM+Fell+French+Canon&family=Montserrat:wght@200&display=swap');
+.blog-title{
+
+    font-family: 'IM FELL French Canon';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 36px;
+    line-height: 35px;
+    color: #000000;
+}
+.blog-sub-title {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 35px;
+    letter-spacing: 0.1em;
+    color: #000000;
+}
+
+#link-color{
```

Comment:
if some of these text styles can actually be common and used in other components, i would recommend defining them as global configs in tailwind for others to share too (this is only fine if it is for a specific part only)

Reply:
and idk why you had to use an ID for this, i would just stick with class names